README.md

```diff
@@ -25,6 +25,8 @@ export PATH=<Your Toolchain PATH>:$PATH
 ```
 Be aware that recommanded toolchain is fully working on 64bits machine.
 
+NOTE: See [Advanced Environments](docs/InstallAdvEnv.md) to use gRPC, ROMFS, IoTivity, IoTjs and so on.
```

Comment:
Additional requirements might be better than 'advanced environments':
Additional requirements for gRPC, ROMFS, IoTivity, and IoT.js.
Please remove 'and so on'.

Reply:
When you say IoT.js, don't forget the dot between 'IoT' and 'js'.

Reply:
Regarding feature-wise description, OCF is the proper term instead of IoTivity.